config/requirements/base.pip

```diff
@@ -23,6 +23,7 @@ mimeparse==0.1.3
 mock==2.0.0
 jsonfield==1.0.3
 psycopg2==2.6.2
+fake-factory==0.7.4
```

Comment:
@strycore 
I had to add this in order to fix `ImportError: The ``fake-factory`` package is now called ``Faker``.` error.
Any Idea why?